monitoring/tests/system/test_vpcsc_v3.py

```diff
@@ -343,26 +311,7 @@ def test_list_monitored_resource_descriptors(self):
         delayed_outside = lambda: client.list_monitored_resource_descriptors(
             name_outside
         )
-        TestVPCServiceControlV3._do_test(delayed_inside, delayed_outside)
-
-    @pytest.mark.skipif(
-        PROJECT_INSIDE is None, reason="Missing environment variable: PROJECT_ID"
-    )
-    @pytest.mark.skipif(
-        PROJECT_OUTSIDE is None,
-        reason="Missing environment variable: GOOGLE_CLOUD_TESTS_VPCSC_OUTSIDE_PERIMETER_PROJECT",
-    )
-    def test_list_time_series(self):
-        client = monitoring_v3.MetricServiceClient()
-        name_inside = client.project_path(PROJECT_INSIDE)
-        delayed_inside = lambda: client.list_time_series(
-            name_inside, "", {}, enums.ListTimeSeriesRequest.TimeSeriesView.FULL
-        )
-        name_outside = client.project_path(PROJECT_OUTSIDE)
-        delayed_outside = lambda: client.list_time_series(
-            name_outside, "", {}, enums.ListTimeSeriesRequest.TimeSeriesView.FULL
-        )
-        TestVPCServiceControlV3._do_test(delayed_inside, delayed_outside)
```

Comment:
~~Why does this PR delete this test?~~ I see that it is moved below:  is there a reason for reordering the testcases here?

Reply:
I've changed the script to arrange tests in a deterministic order by sorting endpoint versions, then services, then methods. Should be good now.